### PR TITLE
Arm refactor code

### PR DESCRIPTION
**Pull request by selym3**

### CODE REVIEW

**Review by vincentw921 — APPROVED** [review]

**Review by anivanchen — APPROVED** [review]

File: src/main/java/com/stuypulse/robot/subsystems/arm/Arm.java [hunk 1]
```diff
@@ -63,24 +54,31 @@ public final ArmState getTargetState() {
...
+    // Read error between measured and target states
     public final boolean isShoulderAtAngle(Rotation2d maxError) {
         return Math.abs(getShoulderTargetAngle().minus(getShoulderAngle()).getDegrees()) < maxError.getDegrees();
     }
     public final boolean isWristAtAngle(Rotation2d maxError) {
         return Math.abs(getWristTargetAngle().minus(getWristAngle()).getDegrees()) < maxError.getDegrees();
     }
 
-    public final boolean isArmAtState(Rotation2d shoulderEpsilon, Rotation2d wristEpsilon) {
+    public final boolean isArmAtCurrentState(Rotation2d shoulderEpsilon, Rotation2d wristEpsilon) {
```
Comment (BenG49) [inline src/main/java/com/stuypulse/robot/subsystems/arm/Arm.java]: rename to isArmAtTargetState

File: src/main/java/com/stuypulse/robot/subsystems/arm/Arm.java [hunk 2]
```diff
@@ -63,24 +54,31 @@ public final ArmState getTargetState() {
...
         return isShoulderAtAngle(shoulderEpsilon) && isWristAtAngle(wristEpsilon);
     }
 
-    public final boolean isArmAtTargetState() {
+    public final boolean isArmAtTargetState(Rotation2d shoulderEpsilon, Rotation2d wristEpsilon) {
```
Comment (BenG49) [inline src/main/java/com/stuypulse/robot/subsystems/arm/Arm.java]: rename to isArmAtEndState?

**Review by BenG49 — APPROVED** [review]
BOOM‼️

**Review by BenG49 — APPROVED** [review]
BOOM‼️

**Pull request merged**